functions/fetch_action_item.ts

```diff
@@ -0,0 +1,74 @@
+import { DefineFunction, Schema, SlackFunction } from "deno-slack-sdk/mod.ts";
+import { queryActionItemDatastore } from "../datastores/action_list_datastore.ts";
+import { ActionItemInfo } from "../types/action_item_info.ts";
+
+export const FetchUserActionItemsFunction = DefineFunction({
```

Comment:
there is a disconnect for the function names and file names, can you align all of them?